pkgs/applications/version-management/git-and-tools/qgit/default.nix

```diff
@@ -16,7 +17,7 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
```

Comment:
You can drop this as cmake enables it by default.

Reply:
Nice feature! When was that introduced?

Reply:
Accoring to ```git blame```:

```
031138b7f02a (Orivej Desh            2017-12-02 06:20:07 +0000 110)     if ! [[ -v enableParallelBuilding ]]; then
031138b7f02a (Orivej Desh            2017-12-02 06:20:07 +0000 111)         enableParallelBuilding=1
031138b7f02a (Orivej Desh            2017-12-02 06:20:07 +0000 112)         echo "cmake: enabled parallel building"
031138b7f02a (Orivej Desh            2017-12-02 06:20:07 +0000 113)     fi
```

Reply:
Thanks! That was quite a while ago. Good to know that this line is not necessary anymore for cmake derivations.